tests/longevity-2mv-backpressure-4d.yaml

```diff
@@ -1,75 +0,0 @@
-test_duration: 6480
```

Comment:
remove the the old test files... that's a very good idea, we should start doing more

Reply:
There is staging Jenkins job with this profile that runs from my private branch. I am going to create new job (not staging) with pipeline. So it's not used and we don't need it